notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -938,3 +938,47 @@ human vs. beast [Paul and Lady Jessica advanced across the desert sands with utm
 what if I had to fight to the death [Paul bested the Fremen warrior Jamis in a ritual knife duel to the death.]
 
 
+movie: Encounter (2021)
+=======================
+
+:: Title
+Encounter
+
+:: Description
+Encounter is a 2021 drama thriller film, directed by Michael Pearce from a
+screenplay by Pearce and Joe Barton. It stars Riz Ahmed, Octavia Spencer,
+Janina Gavankar, Rory Cochrane, Lucian-River Chauhan, and Aditya Geddada.
+
+Synopsis: A former Marine abducts his two young sons to protect them from
+what he perceives to be an imminent threat from space parasites. 
+
+:: Date
+2021-09-03
+
+:: References
+https://en.wikipedia.org/wiki/Encounter_(2021_film)
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+father and son [The story explores how the boy Jay comes to understand that his estranged father is not the hero that he'd believed him to be.]
+child abduction [This is a story about well-intentioned, but misguided father abducting his two young sons to protect them from a threat that exists solely in his imagination.]
```

Comment:
Missing indefinite article before "well-intentioned"

Reply:
Indeed. Fixed and committed.